generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -41,9 +41,8 @@ jhipster.security.authentication.jwt.token-validity-in-seconds-for-remember-me=2
 jhipster.mail.base-url=http://127.0.0.1:8080
 jhipster.info.swagger.enable=true
 
-quarkus.http.auth.permission.auth.paths=/api/authenticate/,/api/register,/api/activate,/api/account/reset-password/init,/api/account/reset-password/finish,/management/health,/management/info,/management/prometheus
-quarkus.http.auth.permission.auth.policy=permit
-quarkus.http.auth.permission.auth.methods=POST
+quarkus.http.auth.permission.public.paths=/api/authenticate,/api/register,/api/activate,/api/account/reset-password/init,/api/account/reset-password/finish,/management/health,/management/info,/management/prometheus
```

Comment:
Should `/management/info,/management/prometheus` be public by default? Could be the case that they are supposed to be but it seems like those might expose more data than desired by default. Want to double check before we merge.

Reply:
they are in public in vanilla jhipster application, so i made them public

Reply:
For reference, in vanilla JHipster apps:

- The `/management/info` endpoint returns data used by the client (for example, the development ribbon). 
- The `/management/prometheus` endpoint is public, but disabled in `prod` by default (see [`application-prod.yml#L22-L26`](https://github.com/jhipster/jhipster-sample-app/blob/8b899a834a5f1033411ecd231bf6f9a2b80fa597/src/main/resources/config/application-prod.yml#L22-L26)).  

And yes the Prometheus endpoint could expose data, which is why it's disabled by default for `prod`. There is information to add basic authentication [in the docs](https://www.jhipster.tech/monitoring/#configuring-metrics-forwarding) but this may need to be adapted for Quarkus.

Reply:
Cool thanks.